airflow/utils/serve_logs.py

```diff
@@ -38,6 +40,15 @@
 logger = logging.getLogger(__name__)
 
 
+def has_dualstack_ipv6():
+    """
+    Method to check if dual stack is supported was added in Python 3.8
+    """
+    if hasattr(socket, 'has_dualstack_ipv6'):
+        return socket.has_dualstack_ipv6()
+    return False
```

Comment:
Since this function is used only once, it’s probably easier to just do

```python
if getattr(socket, "has_dualstack_ipv6", bool)():
    ...
```

instead.

Reply:
Sure, changed